app/controllers/api/v1/featured_tags/suggestions_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Api::V1::FeaturedTags::SuggestionsController < Api::BaseController
+  before_action -> { doorkeeper_authorize! :read, :'read:featured_tags' }, only: [:index]
+
+  before_action :require_user!
+
+  respond_to :json
+
+  def index
+    most_used_tags = Tag.most_used(current_account).where.not(id: current_account.featured_tags).limit(10)
```

Comment:
For code style, move this assignment to a `before_action`

Reply:
Correct the code style.